### ISSUE TITLE
Create an api to get issue of a github repo

### ISSUE THREAD

**Issue by Hasnayeen** [iluminar/goodwork#642]
This api should get open issues from the connected github repo, `github_repos` table contain connected github repo to a group (project/team/office)

**Comment by jonesdev** [comment]
I would like to pick this up tomorrow if no one else gets on this, if that's okay?

Thanks!

**Comment by Hasnayeen** [comment]
Sure, go ahead.

**Comment by SidneyMachara** [comment]
hi , id like to work on this.  just to make sure i understand the requirements --> you need an api of the form api/open_issues/{entity_id}  , which returns all open issues from the github repo that entity_id matches in github_repos ?

**Comment by Hasnayeen** [comment]
@SidneyMachara yes that's correct. here `entity_id` represents one of these (project, team, office), so the api will check if there is any repo is connected to that entity and then fetch all open issues for that repo from github via graphql api. `github_repo_id` is the id of the repo returned from github graphQl api
